central/enrichment/integration.go

```diff
@@ -35,28 +35,42 @@ type managerImpl struct {
 // It loops through the categories, which is a very small slice.
 func isNodeIntegration(integration *storage.ImageIntegration) bool {
 	for _, category := range integration.GetCategories() {
+		log.Infof("isNodeintegration for integration %s: Node_Scanner %v", integration.GetName(), category == storage.ImageIntegrationCategory_NODE_SCANNER)
 		if category == storage.ImageIntegrationCategory_NODE_SCANNER {
 			return true
 		}
 	}
 	return false
 }
 
-// imageIntegrationToNodeIntegration converts the given image integration into a node integration.
-// Currently, only StackRox Scanner is a supported node integration.
-// Assumes integration.GetCategories() includes storage.ImageIntegrationCategory_NODE.
-func imageIntegrationToNodeIntegration(integration *storage.ImageIntegration) (*storage.NodeIntegration, error) {
-	if integration.GetType() != scannerTypes.Clairify {
-		return nil, errors.Errorf("requires a %s config: %q", scannerTypes.Clairify, integration.GetName())
+// ImageIntegrationToNodeIntegration converts the given image integration into a node integration.
```

Comment:
For reviewers: I made this function public to tackle code duplication. 
The identical function existed in `central/imageintegration/service/convert.go`.